src/main/java/com/example/bankservice/repository/UserRepository.java

```diff
@@ -0,0 +1,9 @@
+package com.example.bankservice.repository;
+
+import com.example.bankservice.model.User;
+import java.util.Optional;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+public interface UserRepository extends JpaRepository<User, Long> {
+    Optional<User> findByPhoneNumber(String phoneNumber);
```

Comment:
please see checklist - we should fetch roles along with user in this method and in `findById()` as well